.github/workflows/image-health-check.yml

```diff
@@ -42,27 +42,33 @@ jobs:
       fail-fast: false
       matrix:
         TARGET_PLATFORM:
-          - centos,7
+          # - centos,7
+          # removing due to EOL
           - oraclelinux,8
-          - oraclelinux,7
+          #- oraclelinux,7
+          # removing due to EOL
           - almalinux,8
           - almalinux,9
         POSTGRES_VERSION:
-          - 11
+          # - 11
+          # removing Postgres 11 support due to EOL
           - 12
           - 13
           - 14
           - 15
           - 16
+          - 17
         include:
-          - TARGET_PLATFORM: debian,buster
+          # - TARGET_PLATFORM: debian,buster
+          # removing due to EOL of debian buster
```

Comment:
Please delete all comments for dropped os and postgres support

Reply:
Fixed for all pipelines